packages/webpack/src/utils/postcss.js

```diff
@@ -28,6 +28,17 @@ export const orderPresets = {
   }
 }
 
+const postcssConfigFileWarning = (() => {
```

Comment:
We can in the future probably assign a unique code to the warnings so it can be both used to report them once and also link to the docs. Like https://nuxtjs.org/KB/107

Reply:
@manniL What about storing state in function object? (And a normal function)

```js
function postcssConfigFileWarn() {
  if (postcssConfigFileWarn.executed) {
    return
  }
  consola.warn(`Please use \`build.postcss\` in your nuxt.config.js instead of an external config file. Support for such files will be removed in Nuxt 3 as they remove all defaults set by Nuxt and can cause severe problems with features like alias resolving inside your CSS.`)
  postcssConfigFileWarn.executed = true
}
```

Reply:
> We can in the future probably assign a unique code to the warnings so it can be both used to report them once and also link to the docs. Like https://nuxtjs.org/KB/107

Like the idea for the future!

Reply:
> @manniL What about storing state in function object? (both normal and arrow can do this)

Didn't know about that TBH :thinking: 	
Do you think that approach would make it more readable?

Reply:
Actually, it took me a few seconds to understand what that IIFE does. I think yes, it can probably increase readability and scalability (for multiple warning handling). 

Perf is not a valid concern but it also reduces one function creation :)

Reply:
Would you mind adding the change then as you suggested it? :relaxed:

Reply:
Sure :)

Reply:
@manniL Does it looks good to you now?

Reply:
Yup, LGTM :relaxed: